pkg/arvo/sys/zuse.hoon

```diff
@@ -5453,6 +5453,35 @@
     |=  cis/coin
     ?.  ?=({$$ case} cis)  ~
     `(unit beam)`[~ [who dex `case`p.cis] (flop t.t.t.pax)]
+  ::
+  ++  json-rn                                           ::  json to rn parser
```

Comment:
It's kinda silly to put this parser under `++format`, if someone figures out a better place to put it let me know. Doesn't feel great to duplicate it under ´ne:dejs-soft:format` and `ne:dejs:format` either.